File: Wright_Will/Assignments/Bubble_Sort.py

def bubbleSort(lst):
    for n in range(len(lst)):
        for i in range(1,len(lst)):
            if lst[i-1] > lst[i]:
               temp = lst[i-1]
               lst[i-1] = lst[i]
               lst[i] = temp
    return lst
a = [2,6,4,9,12,5,3,7]

print(bubbleSort(a))

File: Wright_Will/Assignments/BubbleSort.js
function bubbleSort(arr){
    for (var n = 0; n < arr.length; n++) {
        for (var i = 1; i < arr.length; i++) {
            if (arr[i-1]>arr[i]) {
                temp = arr[i-1]
                arr[i-1] = arr[i]
                arr[i] = temp
            }
        }
    }
    return arr
}

var a = [1,5,8,12,0,4,8]
